actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -1,5 +1,4 @@
 require 'abstract_unit'
-require 'digest/sha1'
```

Comment:
@kaspth This requirement looks like some old stuff, I haven't found it used in this test currently (also it's green without it)